### PR TITLE
Added Caldigit Docking Utility v1.9.11

### PR DESCRIPTION
**Pull request by jstader**
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not [already refused].
- [X] Checked the cask is submitted to [the correct repo].

### CODE REVIEW

**Comment by jstader** [comment]
It appears the curl command doesn't work for Xcode 10.1. Does anyone know an easy way around this or a way to require Xcode 10.3?

File: Casks/caldigit-docking-utility.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'caldigit-docking-utility' do
+  version '1.9.11'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/caldigit-docking-utility.rb]: `sha256` also needs to be set. You can find what it is by running `shasum --algorithm 256` on the downloaded file.

Comment (jstader) [reply Casks/caldigit-docking-utility.rb]: I changed the sha to nocheck in an attempt to fix the curl problem. The error still exists with the correct sha sum

Comment (jstader) [reply Casks/caldigit-docking-utility.rb]: There’s some error in Xcode 10.1 that isn’t allowing the file to be downloaded

Comment (vitorgalvao) [reply Casks/caldigit-docking-utility.rb]: There’s no relation between curl and the shasum. Ignore that error; it’s external to the cask.

Comment (jstader) [reply Casks/caldigit-docking-utility.rb]: Got it. I added it back.

**Comment by vitorgalvao** [comment]
It’s now getting a

> Error: pkg source file not found: 'CalDigit Docking Station Utility v1.9.11.pkg'

Meaning the version must have been updated.

**Pull request merged**